### PR TITLE
doc(zh-CN) - update Why Astro Translation

### PR DESCRIPTION
**Pull request by GrainStar**
Translated the English part. But need to check the internal text link situation (I can't get this).

#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)
- New or updated content
- Translated content
- Changes to the docs site code
- Something else!

#### Description

- Closes # 
- What does this PR change? Give us a brief description.
- Did you change something visual? A before/after screenshot can be helpful.

### CODE REVIEW

**Comment by kecrily** [comment]
Please add Han-West space (space between Chinese and English, numbers)

**Review by delucis — COMMENTED** [review]
Thank you for this PR @GrainStar! It looks like the issue is that the file you added is at `src/pages/zh-cn/core-concepts/why-astro.md`, but should be `src/pages/zh-cn/concepts/why-astro.md`. I think renaming it should be all you need to fix those links! (Sorry for our confusing folder names!)

By the way, if you use Discord, we would be happy to have you join us in our #docs-i18n channel where we have dedicated threads for all languages and translators can discuss: https://astro.build/chat

**Comment by Yan-Thomas** [comment]
Thanks for this @GrainStar, there are a few broken links, could you please fix them? Note: You have to change links from `/en/` to `/zh-cn/`. In "Files changed" GitHub gives you an indication of which links are broken and the existing anchors in the page to make it easier for you to fix!

File: src/pages/zh-cn/concepts/why-astro.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+description: "Astro是集多功能于一体的Web框架，用于建立快速、以内容为中心的网站。了解更多。"
+i18nReady: true
+---
+
+Astro 是**集多功能于一体的Web框架**，用于构建快速、以内容为中心的网站。
+
+为什么选择 Astro 而不是其他的Web框架？以下的五个核心设计原则有助于解释我们为什么要构建 Astro，它需要解决的问题以及为什么 Astro 可能是您的项目或团队的最佳选择。
+
+#### Astro是...
+1. [ **以内容为中心** ](#以内容为中心)：Astro 专为内容丰富的网站而设计。
```
Comment (delucis) [inline src/pages/zh-cn/concepts/why-astro.md:13]:
```suggestion
1. [ **以内容为中心** ](#以内容为中)：Astro 专为内容丰富的网站而设计。
```

File: src/pages/zh-cn/concepts/why-astro.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+
+## 服务器优先
+**Astro 尽可能利用服务器渲染而不是客户端渲染。** 这与传统服务器端框架（PHP、WordPress、Laravel、Ruby on Rails等）使用的方法相同，您不需要学习第二种服务端语言。 Astro 仍然使用 HTML、CSS和JavaScript(或TypeScript)。
+
+这种方法与其他现代 JavaScript Web框架 形成鲜明对比，如 Next.JS、SvelteKit、Nuxt、Remix 等。这些框架需要整个网站的客户端和服务器端渲染，以解决性能问题，这种方法被称为**单页应用程序(SPA)，** 与 Astro 的**多页应用程序（MPA）** 方式形成鲜明对比。
+
+SPA模式有它的优势。然而，这些都是以牺牲额外的复杂性和性能权衡为代价，这些权衡会损坏页面性能——包括[可交互时间(TTI)]() 等关键指标——这对于以内容为中心的网站没有多大意义，因为这些网站的首次加载性能至关重要。
+
+📚 [进一步了解 **Astro MPA** 架构的独特之处](/en/concepts/mpa-vs-spa/)
```
Comment (delucis) [inline src/pages/zh-cn/concepts/why-astro.md:38]:
```suggestion
📚 [进一步了解 **Astro MPA** 架构的独特之处](/zh-cn/concepts/mpa-vs-spa/)
```

File: src/pages/zh-cn/concepts/why-astro.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+我们最喜欢的谚语之一是：**选择加入复杂性。** 我们设计 Astro 是为了尽可能多地从开发人员体验中消除“所需的复杂性”，尤其是您首次加入时。您可以在 Astro中使用 HTML和CSS构建“Hello World”示例网站。然后当您需要构建更强大的功能时，您可以随时获得新功能和API。
+
+
+
+
+## 功能齐全且灵活
+
+**Astro 是集多功能于一体的Web框架，提供了构建网站所需的一切** Astro 包括组件语法、基于文件的路由、资产处理、构建处理、捆绑、优化、数据获取等。您可以在不超过 Astro核心功能集 的情况下构建出色的网站。
+
+如果您需要更多的控制，你可以通过 [React](https://www.npmjs.com/package/@astrojs/react), [Svelte](https://www.npmjs.com/package/@astrojs/svelte), [Vue](https://www.npmjs.com/package/@astrojs/vue), [Tailwind CSS](https://www.npmjs.com/package/@astrojs/tailwind), [MDX](https://www.npmjs.com/package/@astrojs/mdx), [image optimizations](https://www.npmjs.com/package/@astrojs/images)等[100多个集成](https://astro.build/integrations/) 扩展 Astro 只需要一个命令 [即可连接您喜欢的CMS](https://astro.build/integrations/) 或 [部署到您喜欢的服务器](/en/guides/deploy/)
```
Comment (delucis) [inline src/pages/zh-cn/concepts/why-astro.md:78]:
```suggestion
如果您需要更多的控制，你可以通过 [React](https://www.npmjs.com/package/@astrojs/react), [Svelte](https://www.npmjs.com/package/@astrojs/svelte), [Vue](https://www.npmjs.com/package/@astrojs/vue), [Tailwind CSS](https://www.npmjs.com/package/@astrojs/tailwind), [MDX](https://www.npmjs.com/package/@astrojs/mdx), [image optimizations](https://www.npmjs.com/package/@astrojs/images)等[100多个集成](https://astro.build/integrations/) 扩展 Astro 只需要一个命令 [即可连接您喜欢的CMS](https://astro.build/integrations/) 或 [部署到您喜欢的服务器](/zh-cn/guides/deploy/)

```

**Pull request closed without merging**